osi_sensorviewconfiguration.proto

```diff
@@ -752,27 +752,126 @@ message CameraSensorViewConfiguration
         //
         CHANNEL_FORMAT_RGB_F32_LIN = 9;
 
-        // Bayer RGGB Channels UINT8 FP Linear.
+        // Bayer BGGR Channels UINT8 FP Linear.
         //
         CHANNEL_FORMAT_BAYER_BGGR_U8_LIN = 10;
 
-        // Bayer RGGB Channels UINT16 FP Linear.
+        // Bayer BGGR Channels UINT16 FP Linear.
         //
         CHANNEL_FORMAT_BAYER_BGGR_U16_LIN = 11;
 
-        // Bayer RGGB Channels UINT32 FP Linear.
+        // Bayer BGGR Channels UINT32 FP Linear.
         //
         CHANNEL_FORMAT_BAYER_BGGR_U32_LIN = 12;
 
-        // Bayer RGGB Channels Single Precision FP Linear.
+        // Bayer BGGR Channels Single Precision FP Linear.
         //
         CHANNEL_FORMAT_BAYER_BGGR_F32_LIN = 13;
 
+        // Bayer RGGB Channels UINT8 FP Linear.
+        //
+        CHANNEL_FORMAT_BAYER_RGGB_U8_LIN = 14;
+
+        // Bayer RGGB Channels UINT16 FP Linear.
+        //
+        CHANNEL_FORMAT_BAYER_RGGB_U16_LIN = 15;
+
+        // Bayer RGGB Channels UINT32 FP Linear.
+        //
+        CHANNEL_FORMAT_BAYER_RGGB_U32_LIN = 16;
+
+        // Bayer RGGB Channels Single Precision FP Linear.
+        //
+        CHANNEL_FORMAT_BAYER_RGGB_F32_LIN = 17;
+
+        // Read Clear Clear Clear Channels UINT8 FP Linear.
```

Comment:
Shouldn't all of those be red (not read) clear...?

Reply:
Changed to red